homeworks/vladislav.buzunar_buzyaka/5-arrays/arrays.js

```diff
@@ -0,0 +1,38 @@
+function christmasTree(height) {
```

Comment:
1) The function name should have a verb. For example, `updateUser`, `fetchEntity`, `findLowestNumber`, `showTicket`, `displayName`.

`christmasTree` is a bad name for a function, because it doesn't describe what the function does and there is no verb in the name. Please, find a new one.

Reply:
Okay, boss.
Understood, i done!